### ISSUE TITLE
expose isXXXChanging properties on LocalDateTimePicker

### ISSUE THREAD

**Issue by millmanorama** [JFXtras/jfxtras#6]
I need to know when a user has chosen a new LocalDateTime, but not be notified of every intermediate LocalDateTime before the user dismisses the popup.  It would be nice if a isChanging property were exposed like for sliders.

**Comment by tbee** [comment]
You are talking about the LocaleDateTimeTextField I assume. And you want to know each date/time combination that was selected. The problem with that is that using the time sliders, each position / time the cursor halts on (but the mouse is not released) is considered a selected time. So this will result in a lot of "isChanging" times...

Tom

**Comment by millmanorama** [comment]
sorry, I missed a 'not'.  I don't want to be notified of all the intermediate times selected with the slider.  Currenlty, each time the mouse stops but is not released i get a change notification on the selected date/time, but i have no way of knowing if that is an intermediate one or not(as far as i can tell).  I thought if a isChanging property was exposed like for a slider i could check this in my handler and ignore spurious notifications of non final selections.

**Comment by tbee** [comment]
Ok. This continuous updating was in fact something that I ignored; didn't like, but it wasn't incorrect either. Your remark finally triggered me to look into it. I've decided to hide the "is changing" events; so the value is only set after the mouse is released. The intermediate events only update the screen. I do not want to expose how the UI renders the time and that coincidentally the slider has all these "while changing" events.
